goblin/engine.py

```diff
@@ -43,7 +41,7 @@
     msg = await stream.fetch_data()
     features['threaded_transactions'] = msg.data[0]
 
-    return Engine(url, conn, loop, **features)
+    return Engine(url, conn, loop, force_close=force_close, **features)
```

Comment:
I think I remember talking about connections and responses having `force_close` before we switched to the queue-based responses and multiplexed connections. What does it mean for an Engine to be force_close? Does it execute one request/response cycle and then self-destruct?